### PR TITLE
Chevaun Ho Assignment 1 Week 1

### PR DESCRIPTION
**Pull request by chevaunho**
Two pages added (resume and contact me) and added to nav bar.
Embedded Instagram feed.
Problems with favicon display.

### CODE REVIEW

File: Week_01_Basics/Assignment/stylesheet.css [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+	margin: 0px 0px 0px 0px;
+	padding: 0px 0px 0px 0px;
+}
+
+nav {
+	padding: 10px;
+	text-align: center;
+}
+ul {
+    list-style-type: none;
```
Comment (yangli-io) [inline Week_01_Basics/Assignment/stylesheet.css:15]: Inconsistent indentation levels

File: Week_01_Basics/Assignment/resume.html [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+</head>
+<br/>
+<hr>
+<br/>
+<body>
+	<table>
+	  <tr>
+	  	<td>
+			<h2>Educational achievements...</h2>
+				<p>&middot Front End Web Development, General Assembly </p>
```
Comment (yangli-io) [inline Week_01_Basics/Assignment/resume.html:31]: You can use a list here to achieve the middot.  Otherwise the middot is correctly used as `&middot;`

Comment (chevaunho) [inline Week_01_Basics/Assignment/resume.html:31]: I wasn't sure how to override the settings in css as I needed a specific style ul for the nav bar.

Comment (yangli-io) [inline Week_01_Basics/Assignment/resume.html:31]: That's fine then, just need to add the semi-colon at the end of &middot

File: Week_01_Basics/Assignment/portfolio.html [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+<head>
+	<header>
+		<h1>Chevaun&middotHo</h1>
+		<h4>SYDNEY-BASED CREATIVE</h4>
+		<h3>painter &middot journalist &middot social media producer &middot front end web developer</h3>
+
+		<!-- Site navigation menu -->
+		<nav>
+			<div class="navbar">
+				<ul>
```
Comment (yangli-io) [inline Week_01_Basics/Assignment/portfolio.html:15]: I dont think you've closed this `<ul>`

**Comment by yangli-io** [comment]
Great work

**Pull request closed without merging**